// Copyright 2013 The Chromium Authors. All rights reserved.
// Use of this source code is governed by a BSD-style license that can be
// found in the LICENSE file.
//
// Protocol buffer definition for a DomDistiller entry.

syntax = "proto2";

option optimize_for = LITE_RUNTIME;

package dom_distiller;

message ArticleEntry {
  // Next ID to use: 3

  optional string entry_id = 1;

  message Page {
    // Next ID to use: 2

    optional string url = 1;
  }

  repeated Page pages = 2;
}

